### DISCUSSION TITLE
WYSIWYG Editor Component

### DISCUSSION THREAD

**Discussion by FortyGazelle700** [saadeghi/daisyui#2475] [Ideas / feature requests]
I think it would be awesome to have a WYSIWYG Editor Component to be able to submit forms, and about information in apps using Daisy UI, I love the UI library and would love to see a component like this added!

**Comment by NikarashiHatsu** [comment]
A WYSIWYG editor is purely a JavaScript component which is impossible to achieve in a CSS library.

**Comment by rikgirbes** [comment]
I just use TipTap editor, where the toolbar can just be created with DaisyUI buttons.
Same goes for a border/shadow/background (base-100/300).
Then the editor just goes along with everything else in the current theme.

https://tiptap.dev/
